--- langchain_mongodb-0.11.0.orig/pyproject.toml	2026-06-02 16:32:00
+++ langchain_mongodb-0.11.0/pyproject.toml	2026-06-02 16:32:00
@@ -11,7 +11,6 @@
 requires-python = ">=3.10"
 dependencies = [
     "langchain-core>=1.2.5",
-    "langchain>=1.0",
     "langchain-classic>=1.0",
     "pymongo>=4.6.1",
     "langchain-text-splitters>=1.0",
@@ -23,7 +22,6 @@
 [dependency-groups]
 dev = [
     "freezegun>=1.2.2",
-    "langchain>=1.0",
     "langchain-core>=1.2.5",
     "langchain-text-splitters>=1.0",
     "pytest-mock>=3.10.0",
